### DISCUSSION TITLE
Laravel Vuejs project how to use Index.php instead of index.html?

### DISCUSSION THREAD

**Discussion by stephenjason89** [ionic-team/capacitor#3577] [Q&A] [answered]
I'm very new to capacitor. Would really appreciate if someone can point me to the right direction.
Laravel / vue project has an index.php inside the public folder

I am getting this error
```
The web directory (C:\Development\Web Projects\Foodat\public) must contain a "index.html".
    It will be the entry point for the web portion of the Capacitor app.
```

How would i configure capacitor to accept index.php?

thank you

**Comment by jcesarmobile** [comment, marked as answer]
You can't, PHP only works on a server and Capacitor can't run a PHP server inside.

The only way is pointing to a remote PHP server hosting the index.php, something like this, but it's not really recommended to point to an url in production apps and it can lead to app rejections.
 
```
"server": {
  "url": "https://url.to.a.php.server/index.php"
}
```

**Reply by stephenjason89** [reply]
Thank you for the reply, I created a nuxt project for my front end and migrated everything.
Separated my laravel backend.
Now everything is working.

By the way, do you know of any way i can build the ios app on windows machine?

**Reply by jcesarmobile** [reply]
You can't build iOS apps from windows machine, you need to use some CI/CD platform or online build service such as [Appflow](https://ionicframework.com/appflow), macincloud, etc.

**Comment by stephenjason89** [comment]
Thank you :)
